tiled/server/authentication.py

```diff
@@ -912,6 +912,35 @@ async def principal(
     )
 
 
+@base_authentication_router.delete(
+    "/principal/{uuid}/apikey",
+    response_model=schemas.Principal,
+)
+async def delete_service_principal_apikey(
```

Comment:
We have methods named:

````
new_apikey
revoke_apikey
apikey_for_principal
```

so maybe this should be `revoke_apikey_for_principal`? Certainly the word `service` shouldn't be in here because it can be used to revoke other users' API keys as well, as the docstring correctly says.